include/crm/common/output.h

```diff
@@ -24,23 +26,23 @@ extern "C" {
  * \brief Control which sections are output
  */
 typedef enum {
-    pcmk_section_stack         = 1 << 0,
-    pcmk_section_dc            = 1 << 1,
-    pcmk_section_times         = 1 << 2,
-    pcmk_section_counts        = 1 << 3,
-    pcmk_section_options       = 1 << 4,
-    pcmk_section_nodes         = 1 << 5,
-    pcmk_section_resources     = 1 << 6,
-    pcmk_section_attributes    = 1 << 7,
-    pcmk_section_failcounts    = 1 << 8,
-    pcmk_section_operations    = 1 << 9,
-    pcmk_section_fence_failed  = 1 << 10,
-    pcmk_section_fence_pending = 1 << 11,
-    pcmk_section_fence_worked  = 1 << 12,
-    pcmk_section_tickets       = 1 << 13,
-    pcmk_section_bans          = 1 << 14,
-    pcmk_section_failures      = 1 << 15,
-    pcmk_section_maint_mode    = 1 << 16,
+    pcmk_section_stack         = UINT32_C(1) << 0,
```

Comment:
Technically this changes public API (in a way that should not affect behavior on reasonable systems)